packages/flutter_tools/lib/src/dart/package_map.dart

```diff
@@ -13,7 +13,7 @@ import '../base/logger.dart';
 
 const String kPackagesFileName = '.packages';
 
-String get globalPackagesPath => _globalPackagesPath ?? kPackagesFileName;
+String get globalPackagesPathDepreactedNoUse => _globalPackagesPath ?? kPackagesFileName;
```

Comment:
Can we `@Deprecated('Prefer BuildInfo.packagesPath')` instead of hinting in the name?
Edit: I tried this and there were no analyzer warnings where it was used.  Does that annotation not work?

Reply:
Did we disable it for the framework? FYI @goderbauer ?

Reply:
In practice, I tend not to use deprecated because it would flag the framework warning lint

Reply:
That's annoying.